README.md

```diff
@@ -215,9 +220,8 @@ The versioning scheme used is [SemVer AKA Semantic Versioning](https://semver.or
    [this commit](https://github.com/wireapp/core-crypto/commit/e680590ba6969ca2dc47aa38c7f220ea8974b369)
    as a reference.
 1. Push your `release/X.Y.Z` branch and create a PR for it
-1. Get it reviewed, then merge it into `develop` and remove the release branch from the remote.
-1. Now, pull your local develop `git checkout develop && git pull`
-1. Update the `main` branch: `git checkout main && git pull && git merge --ff-only develop`
+1. Get it reviewed, then merge it into `main` and remove the release branch from the remote.
```

Comment:
Do we want to delete release branches?

Reply:
Oh, good catch! No, once we make a release branch (of the form `release/1.x`), that should stay.
But the temporary ones, used for the PR should be removed, it's just that those were also named "release branch". :sweat_smile: 

Will reword that to make it clear, thanks a bunch for noticing this! :purple_heart: